src/app/shared/components/template/components/button/button.component.ts

```diff
@@ -14,25 +42,22 @@ import { TemplateBaseComponent } from "../base";
   styleUrls: ["./button.component.scss"],
 })
 export class TmplButtonComponent extends TemplateBaseComponent implements OnInit {
-  /** TEMPLATE PARAMETER: "style" */
-  style:
-    | "information"
-    | "navigation"
-    | "full"
-    | "flexible"
-    | "medium"
-    | "short"
-    | "tall"
-    | "standard"
-    | "alternative" = "information";
+  /** TEMPLATE PARAMETER: "variant" */
+  variant: IButtonParams["variant"] = "information";
+  /** TEMPLATE PARAMETER: "style". Legacy, use "variant" instead. */
+  style: IButtonParams["style"];
   /** TEMPLATE PARAMETER: "disabled". If true, button is disabled and greyed out */
-  disabled: boolean = false;
+  disabled: IButtonParams["disabled"] = false;
```

Comment:
nit(non-blocking)
it might be easier to use a single `params` variable typed with the definition above to ensure all parameters are covered in the code